CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 ### Internal improvements
 
+#### exonum
+
+- Default implementation of `check` method was added to `Field` trait (#639)
```

Comment:
Please add a dot. :upside_down_face: 

Additionally, the description can be expanded a little: ...method was added to `Field` trait to reduce boilerplate.